lib/ansible/modules/cloud/azure/azure_rm_containerregistry_facts.py

```diff
@@ -130,6 +130,10 @@
                     returned: when registry exists and C(admin_user_enabled) is set
                     type: str
                     sample: pass2value
+        tags:
```

Comment:
The init method is 

https://github.com/ansible/ansible/blob/4ca12e12c3db28e46a9db912e12dcb63f2653cfa/lib/ansible/modules/cloud/azure/azure_rm_containerregistry_facts.py#L177

Reply:
that's correct, because supports_tags is only for main modules, right?